### PR TITLE
Ycb loader refactor

### PR DESCRIPTION
**Pull request by kekeblom**
I tried to clean up the ycb dataloader code as a first exercise. Please do anything you can to make sure I didn't break anything.

To test this I did two things:
- ran `src/loaders_v2/test/loaders_v2_test.py`
- ran `tools/lightning.py`

The images generated by the loaders script all look fine. The training pipeline runs, but the loss is reported as `nan`. I checked and this is the case also on the master branch so it's not necessarily a regression.

### CODE REVIEW

File: src/loaders_v2/dataset_generic.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def __init__(self, cfg_d, cfg_env):
         self._obj_list_sym = cfg_d['obj_list_sym']
         self._obj_list_fil = cfg_d['obj_list_fil']
         self._batch_list = self._backend._batch_list
-        self._force_one_object_visible = cfg_d['output_cfg']['force_one_object_visible']
+        self._force_one_object_visible = cfg_d['output_cfg'].get('force_one_object_visible', False)
```
Comment (JonasFrey96) [inline src/loaders_v2/dataset_generic.py:19]: i didn't know that you can set a default value in the .get method this is beatiful.
